### PR TITLE
importer-rest-api-specs: remove `operations` from tags that we filter out

### LINKED ISSUE #2268: The PostrgeSQLHSC.Cluster.restart API isn't generated to go azure sdk
There is an API for the [restart](https://github.com/Azure/azure-rest-api-specs/blob/09119c0cca116b1d8b69552bb63d697628f0e5f6/specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/stable/2022-11-08/postgresqlhsc.json#L1255) operation of PostgreSQL HyperScale cluster. But seems I didn't find it from [go-azure-sdk](https://github.com/hashicorp/go-azure-sdk/tree/main/resource-manager/postgresqlhsc/2022-11-08). Could you help generate it since I need it to restart cluster after we updated the configuration successfully? Thanks.

### PR DESCRIPTION
**Pull request by stephybun**
Closes  #2268

### CODE REVIEW

File: tools/importer-rest-api-specs/components/parser/swagger_tags_test.go [hunk 1]
```diff
@@ -44,3 +44,69 @@ func TestParsingOperationsUsingTheSameSwaggerTagInDifferentCasings(t *testing.T)
...
+
+	// sanity checking
+	if len(resource.Constants) != 0 {
+		t.Fatalf("expected 0 constants but got %d", len(resource.Constants))
+	}
+	if len(resource.Models) != 1 {
+		t.Fatalf("expected 1 model but got %d", len(resource.Models))
+	}
+	if len(resource.Operations) != 3 {
+		t.Fatalf("expected 4 Operations but got %d", len(resource.Operations))
```
Comment (tombuildsstuff) [inline tools/importer-rest-api-specs/components/parser/swagger_tags_test.go:73]: 3 or 4?

File: tools/importer-rest-api-specs/components/parser/swagger_tags_test.go [hunk 2]
```diff
@@ -44,3 +44,69 @@ func TestParsingOperationsUsingTheSameSwaggerTagInDifferentCasings(t *testing.T)
...
+
+	// sanity checking
+	if len(resourceOperation.Constants) != 0 {
+		t.Fatalf("expected 0 constants but got %d", len(resource.Constants))
+	}
+	if len(resourceOperation.Models) != 1 {
+		t.Fatalf("expected 1 model but got %d", len(resource.Models))
+	}
+	if len(resourceOperation.Operations) != 1 {
+		t.Fatalf("expected 4 Operations but got %d", len(resource.Operations))
```
Comment (tombuildsstuff) [inline tools/importer-rest-api-specs/components/parser/swagger_tags_test.go:102]:
```suggestion
		t.Fatalf("expected 1 Operation but got %d", len(resource.Operations))
```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
Left a couple of comments inline but otherwise 👍

File: tools/importer-rest-api-specs/components/parser/swagger_tags_test.go (hunk 1 above)
Comment (stephybun) [reply tools/importer-rest-api-specs/components/parser/swagger_tags_test.go:73]: 3, because the 4th operation belongs to the 2nd resource.

**Review by tombuildsstuff — APPROVED** [review]

**Pull request merged**